docs/pages/installation/installation.mdx

```diff
@@ -0,0 +1,142 @@
+---
+title: Installing Teleport
+description: How to install Teleport and Teleport's client tools on your platform, including binaries and instructions for Docker and Helm.
+h1: Installation
+videoBanner: fjk29wqXm1A
+tocDepth: 3
+---
+
+This guide shows you how to install Teleport binaries on your platform,
+including:
+- `teleport`
+- `teleport-update`
+- `tsh`
+- `tctl`
+- `tbot`
+- `fdpass-teleport`
+
+If you are new to Teleport, we recommend following our [getting started
+guide](../get-started.mdx).
+
+For best results, Teleport clients (`tsh`, `tctl`, `tbot`) should be the same major
+version as the cluster they are connecting to. Teleport servers are compatible
+with clients that are on the same major version or one major version older.
+Teleport servers do not support clients that are on a newer major version.
+
+See our [Upgrading](../upgrading/upgrading.mdx) guide for more information.
+
+
+## Operating system support
+
+Teleport is officially supported on the platforms listed below. It is worth
+noting that the open-source community has been successful in building and
+running Teleport on UNIX variants other than Linux \[1].
+
+| Operating System | `teleport` Daemon | `tctl` Admin Tool | `tsh` and Teleport Connect User Clients [2] | Web UI (via the browser) | `tbot` Daemon |
+| - | - | - | - | - | - |
+| Linux v2.6.23+ (RHEL/CentOS 7+, Rocky Linux 8+, Amazon Linux 2+, Amazon Linux 2023+, Ubuntu 16.04+, Debian 9+, SLES 12 SP 5+, and SLES 15 SP 5+) \[3] | yes | yes | yes | yes | yes |
+| macOS 11+  (Big Sur)| yes | yes | yes | yes | yes |
+| Windows 10+ (rev. 1607) \[4] | no | yes | yes | yes | yes |
+
+\[1] *Teleport is written in Go and many of these system requirements are due to the requirements
+of the [Go toolchain](https://github.com/golang/go/wiki/MinimumRequirements)*.
+
+\[2] *`tsh` is a Command Line Client (CLI) and Teleport Connect is a Graphical User Interface (GUI) desktop client. See
+  [Using Teleport Connect](../connect-your-client/teleport-connect.mdx) for usage and installation*.
+
+\[3] *Enhanced Session Recording requires Linux kernel v5.8+*.
+
+\[4] *Teleport server does not run on Windows yet, but `tsh`, `tbot`, `tctl`, and Teleport Connect (the Teleport desktop clients)
+support most features on Windows 10 and later.*
+
+## Start Here - Use Cases
```

Comment:
Would it make sense to incorporate this section with the introductory section of this guide?

Also, I would rename the heading. It seems a bit jarring that the text reads "Start here" but it's located midway down the page.

Once we rename the heading, we should use sentence case per the style guide.